javascript/javascript1/week1/js-week1-homework/script.js

```diff
@@ -1,81 +1,112 @@
 function getMyAge() {
-    const yearOfBirth = 1983; //your birth year
-    const yearFuture = 2027; //future year to calculate
-    let age = yearFuture - yearOfBirth; //calculate age
-    //answer result
-    let logString = "You will be " + age + " years old in " + yearFuture; //prepare text to answer
-    
-    alert(logString);
-    console.log(logString);
+  const yearOfBirth = 1983; //your birth year
+  const yearFuture = 2027; //future year to calculate
+  let age = yearFuture - yearOfBirth; //calculate age
+  //answer result
+  let logString = "You will be " + age + " years old in " + yearFuture; //prepare text to answer
+
+  alert(logString);
+  console.log(logString);
 }
 
 function getDogAge() {
-    const dogYearOfBirth = 2017; //Dog birth year
-    const dogYearFuture = 2027; //Future Year
-    let dogYear = dogYearFuture - dogYearOfBirth; //Calculate Age
-    let shouldShowResultInDogYears = true; //Set true if is for dog year and set false for human year only
-    let logString = "Your dog will be " + dogYear + " human years old in " + dogYearFuture; //first is show human year
-    /*
-    check if variable of dogyear is true 
-    then change human year to dog year
-    by add 7 times to human year and
-    add result to string
-    */
-    if (shouldShowResultInDogYears) {
-        dogYear *= 7;
-        logString += "\n or your dog will be " + dogYear + " dog years old in " + dogYearFuture;
-    } 
-    //answer result
-    alert(logString);
-    console.log(logString);
+  const dogYearOfBirth = 2017; //Dog birth year
+  const dogYearFuture = 2027; //Future Year
+  let dogYear = dogYearFuture - dogYearOfBirth; //Calculate Age
+  let shouldShowResultInDogYears = true; //Set true if is for dog year and set false for human year only
+  let logString =
+    "Your dog will be " + dogYear + " human years old in " + dogYearFuture; //first is show human year
+
+  if (shouldShowResultInDogYears) {
+    dogYear *= 7;
+    logString +=
+      "\n or your dog will be " +
```

Comment:
You added a `\n` newline character in a string?? 😲 Magician! This is definitely a way to include the 'dog age' into the string, however, you could have just created two different string for each case of the `if`, and displayed only that. But this is also perfectly okay, and the newline makes it nice and easy to read! 👍